asm/battle_finger_data.s

```diff
@@ -0,0 +1,16 @@
+	.include "asm/macros.inc"
+	.include "overlay_12_0226BBC4.inc"
+	.include "global.inc"
+
+.rodata
```

Comment:
is there any reason why this file isn't included in battle_finger.c? unable to match it?

Reply:
I tried to match it but whatever data is right below it causes an issue, and I can't figure out where it even gets used

Reply:
the sinit? or the rodata?

Reply:
sinit, is that not a form of data? I haven't seen the keyword before so I could be wrong

Reply:
sinit is static initialiser, there are examples of how it's done in diamond